integration/test/basic/main_test.go

```diff
@@ -20,7 +21,10 @@ import (
 )
 
 const (
-	appName = "cert-manager"
+	app            = "cert-manager"
+	appName        = "cert-manager-app"
```

Comment:
The app suffix is a pain here and in other places. See https://github.com/giantswarm/giantswarm/issues/9815